php/class-photonfill-transform.php

```diff
@@ -97,6 +97,12 @@ public function transform_photon_url( $args, $image_url, $scheme = null ) {
 				$args = $this->args;
 			}
 
+			if ( ! empty( $args['resize'] ) ) {
```

Comment:
stick a comment above this as to why. we need more comments in photonfill. this reminds me, i need to go add mine to my PR from yesterday:)